src/nvim/api/window.c

```diff
@@ -432,3 +432,41 @@ Boolean nvim_win_is_valid(Window window)
   return ret;
 }
 
+
+/// Turn window into float, or reconfigure existing float
+///
+/// See documentation at |nvim_open_float_win|, for the meaning of parameters.
+///
+/// When reconfiuring an existing float, absent option keys will not be
+/// changed. The following restriction apply though: `x`, `y` and `relative`
```

Comment:
There is `row` and `col` instead of `x`, `y` at documentation of `nvim_open_float_win`, so what should be used?

Reply:
row, col. I'm in the process of updating it